### PR TITLE
Style refactor

### PR DESCRIPTION
**Pull request by eileenprice**
remove useless style code, move base chart style code into more specific files, create a separate file for styling tooltips

### CODE REVIEW

File: addon/components/row-chart/style.less [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
+
+.d3-tip.e {
+    margin-left: 10px;
+}
+
+.d3-tip.e:after {
+    transform: rotate(90deg);
+    top: 50%;
+    left: -52%;
+}
```
Comment (Jbcampbe) [inline addon/components/row-chart/style.less]: The way these styles are written right know they will apply to all charts. Is that intended?

File: addon/styles/tip.less [hunk 2]
```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700');
```
Comment (Jbcampbe) [inline addon/styles/tip.less]: Looks like we have multiple imports like this in our styles. Maybe we could just include one in `variables.less` or `addon.less`?

File: addon/components/row-chart/style.less (hunk 1 above)
Comment (eileenprice) [reply addon/components/row-chart/style.less]: Yes, currently row chart is the only one that uses eastward tooltips. Tooltips are all actually rendered outside of charts, so you can't force it to apply only to one type of chart unless you add a class to the tooltip, but that shouldn't be necessary here. All eastward tooltips should be rotated like this regardless of the chart. Maybe then this code would be better in `tip.less`?

File: addon/styles/tip.less (hunk 2 above)
Comment (eileenprice) [reply addon/styles/tip.less]: 👍

File: addon/styles/chart.less [hunk 3]
```diff
@@ -7,196 +7,40 @@
...
-        fill: steelblue;
-        fill-opacity: .125;
-    }
-
-    .brush .resize path {
-        fill: @gallery;
-        stroke: @dove-gray;
+    .chart-not-available {
+        text-shadow:
+            -1px -1px 0 #fff,
```
Comment (maiseyh) [inline addon/styles/chart.less]: Super minor but these rows seem a little, spacing-wise. I think this one has 12 spaces and the 3 below have 13.

**Pull request merged**